.github/workflows/Tests.yaml

```diff
@@ -339,6 +341,16 @@ jobs:
           - compiler: clang-9
             build_type: Debug
             use_pch: OFF
+        exclude:
+          # The following tests segfault in release mode:
+          # - Unit.Evolution.Systems.Cce.Equations
+          # - Unit.Evolution.Systems.Cce.InitializeJ
+          # - Unit.Evolution.Systems.Cce.NewmanPenrose
+          # Test_BulgedCube is fixed. This is either a GCC-10 optimizer bug or
+          # some weird Blaze behavior with GCC-10.
```

Comment:
[opinion, you may choose to disagree] Delete this last sentence, which isn't useful to a future reader of the comment

Reply:
I disagree because I think it is useful to forward as much information as I can about what the underlying bug may be. It's not a lot (or really any) information, but I think it's better than none :shrug: I think knowing that the issue is related to Blaze math expressions is useful since I do know that this is why the segfaults are occurring. I was unable to figure out the underlying Blaze & compiler interaction that resulted in the segfault, though.